kpi/explores/unified_metrics_pop_method1.explore.lkml

```diff
@@ -1,13 +0,0 @@
-include: "../views/unified_metrics_pop_method1.view.lkml"
```

Comment:
I couldn't find the `../views/unified_metrics_pop_method1.view.lkml` file. I assume it's already gone then?

Reply:
yes, it is gone, thank you